docs/guides/alias.rst

```diff
@@ -42,6 +44,11 @@ default), ALIAS records are sent as-is (RRType 65401 and a DNSName in
 the RDATA) in the AXFR. When set to 'yes', PowerDNS will lookup the A
 and AAAA records of the name in the ALIAS-record and send the results in
 the AXFR.
+If the ALIAS target can not be resolved during AXFR the AXFR will fail.
+To allow outgoing AXFR also if the ALIAS targets are broken you can set
+:ref:`setting-outgoing-axfr-expand-alias` to 'ignore-errors', but
+be warned, this will lead to inconsistent zones between the Primary and
+Secondary name servers.
```

Comment:
please add some formatting so this warning is *very* clear

Reply:
oh, speaking of reformatting, we're trying to transition to one sentence per line, which makes diff cleaners. Can you do that for the bits you're adding/changing?

Reply:
ah yes, done